examples/desktop/gridplot/gridplot.py

```diff
@@ -22,6 +23,9 @@
 fig[1, 0].add_image(data=im3)
 fig[1, 1].add_image(data=im4)
 
+# set canvas variable for sphinx_gallery to properly generate examples
```

Comment:
specifiy that this line isn't required for users, or see if there's a way for the `pygfx` scraper to look for `fig.canvas` instead of `canvas`? 

in all examples

Reply:
I think perhaps we can use `figure` instead...

`pygfx` scraper looks for one of `["renderer", "canvas"]` to be defined, but will also look through proxy objects such as `"figure", "display", "disp", "plot"` 

So I will try that

Reply:
I think for now, we should just to `canvas = fig.canvas` and have a note saying that it is not required for users and after FENS I can look into fixing them